docs/guides/Package-authoring-best-practices.md

```diff
@@ -0,0 +1,179 @@
+
+## Types of recommendations
+
+Each article presents four types of recommendations: **Do**, **Consider**, **Avoid**, and **Do not**. The type of recommendation indicates how closely it should be followed.
+
+You should almost always follow a **Do** recommendation. For example:
+
+✔️ DO include a short description for your package that describes what it's for.
+
+On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule:
+
+✔️ CONSIDER choosing a NuGet package name with a prefix that meets NuGet's prefix reservation [criteria](https://docs.microsoft.com/nuget/reference/id-prefix-reservation).
+
+**Avoid** recommendations mention things that are generally not a good idea, but breaking the rule sometimes makes sense:
+
+❌ AVOID NuGet package references that demand an exact version.
+
+And finally, **Do not** recommendations indicate something you should almost never do:
+
+❌ DO NOT use the `LicenseUrl` metadata property.
+
+## Create a NuGet package
+
+The latest recommended way to to create a NuGet package is from an [SDK-style project](https://docs.microsoft.com/nuget/resources/check-project-format). SDK-style project properties, including target [framework](#framework-targeting) and [package metadata](#package-metadata), are defined in the project file (.csproj).
+
+Create package from your SDK-style project by defining the required properties and packing in [Visual Studio](https://docs.microsoft.com/nuget/quickstart/create-and-publish-a-package-using-visual-studio?tabs=netcore-cli) or the [dotnet CLI](https://docs.microsoft.com/nuget/quickstart/create-and-publish-a-package-using-the-dotnet-cli).
+
+✔️ DO create an SDK-style project and create (pack) your package using Visual Studio or the dotnet CLI.
+
+For more detailed guidance regarding package creation including necessary client tools, project file example, and commands, see [Create a NuGet package using the dotnet CLI](https://docs.microsoft.com/nuget/create-packages/creating-a-package-dotnet-cli).
+## Framework targeting
+
+Understanding the differences between target frameworks and deciding which one or ones that are right for your library can be very confusing given the wide variety of options available from .NET Framework to .NET 5. So let's try to make it as straightforward as possible.
+
+✔️ CONSIDER starting with [the latest .NET target framework](https://docs.microsoft.com/dotnet/standard/frameworks#latest-versions).
+> Using the latest .NET target framework will allow you to take advantage of the latest .NET technology and language features.
```

Comment:
This section is a bit wordy. Also, I'd avoid mentioning .NET 5 directly as otherwise this will need to be updated in a year :)

Also, I disagree with this recommendation. If you target the latest .NET target framework, .NET Framework folks won't be able to use your new package. Personally I would suggest .NET Standard 2.0 if you want wide support, or latest .NET target framework if you want to use latest features. I don't think there's a cookie cutter answer here, this is a decision you'll need to make as author.

Reply:
Definitely agree w/ not mentioning .NET 5 directly. But we should say "the latest .NET version or future version is the foundation for sharing code". 

https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/

I would just point a link to https://docs.microsoft.com/en-us/dotnet/standard/frameworks#latest-versions as they are in the right order of precedence already.

Reply:
@JonDouglas this section was actually removed entirely based on feedback from Immo. This is all I say now:

![image](https://user-images.githubusercontent.com/15097183/106301406-ae267500-6225-11eb-81fe-db7cd9cc4753.png)